elements/pfe-toast/src/pfe-toast.scss

```diff
@@ -16,7 +16,7 @@ $LOCAL-VARIABLES: (
 
   close: (
     svg--Fill: pfe-var(feedback--default--darkest),
-  	svg--Fill--hover: $pfelements--gray-dark, // #333
+  	svg--Fill--hover: #333, // @TODO: Find $pf parity (was: $pfelements--gray-dark)
```

Comment:
The old $pfelement variables were removed in favor of the pf core variables.  This one didn't have parity so rather than change it, I hardcoded with a TODO.  It was the only reference to that variable so why keep the variable just for this one example?